File: c#/findLargest.cs
using System;
using System.Runtime.InteropServices;

namespace findLargest
{
    class Program
    {
        static void Main(string[] args)
        {
            int[] numbers = [5, 56, 123, 31, 23, 1023, 32, 53];
            Console.WriteLine($"{findLargest(numbers)}");
        }

        static int findLargest(int[] nums)
        {
            int largest = 0;
            for (int i = 0; i < nums.Length; i++){
                if (nums[i] > largest){
                    largest = nums[i];
                }
            }

            return largest;
        }
    }
}

File: python/findLargest.py
class Main:

    # Static methods mean that these can be called without having to create an instance of this class.
    @staticmethod
    def main():
        numbers = [10, 23, 52, 6, 1, 3, 4, 102]
        print(f"largest number: {Main.findLargest(numbers)}")

    @staticmethod
    def findLargest(nums):
        largest = 0
        for i in nums:
            if i > largest:
                largest = i
        return largest

if __name__ == "__main__":
    Main.main()
